ui/src/components/EmailForm.vue

```diff
@@ -7,9 +7,6 @@
         <h3 class="headline mb-0">{{ $t("groups.members.email.compose") }}</h3>
       </div>
     </v-card-title>
-    <!-- TODO: Can we use a loop to render the following 3 components?
-      Currently the issue is that v-model won't synchronize references
-      that are returned by computed properties <2020-06-17, David Deng> -->
```

Comment:
Was this concern addressed?

Reply:
Yes, we figured out that this issue could be solved by coding the 3 components separately.